### ISSUE TITLE
Try lower learning rate

### ISSUE THREAD

**Issue by ChamodKalupahana** [ChamodKalupahana/kaggle_mohs_hardness#9]
Model might be overfitting since lr is too high
